tiledb/sm/c_api/tiledb.h

```diff
@@ -4497,6 +4538,99 @@ TILEDB_EXPORT int32_t tiledb_query_get_fragment_timestamp_range(
     uint64_t* t1,
     uint64_t* t2);
 
+/* ****************************** */
+/*          QUERY CONDITION       */
+/* ****************************** */
+
+/**
+ * Allocates a TileDB query condition object.
+ *
+ * **Example:**
+ *
+ * @code{.c}
+ * uint32_t value = 5;
+ * tiledb_query_condition_t* query_condition;
+ * tiledb_query_condition_alloc(
+ *   ctx, "longitude", &value, sizeof(value), TILEDB_LT, &query_condition);
+ * @endcode
+ *
+ * @param ctx The TileDB context.
+ * @param attribute_name The attribute name.
+ * @param condition_value The value to compare against an attribute value.
+ * @param condition_value_size The byte size of `condition_value`.
+ * @param op The comparison operator.
+ * @param cond The allocated query condition object.
+ * @return `TILEDB_OK` for success and `TILEDB_OOM` or `TILEDB_ERR` for error.
+ */
+TILEDB_EXPORT int32_t tiledb_query_condition_alloc(
+    tiledb_ctx_t* ctx,
+    const char* attribute_name,
+    const void* condition_value,
+    uint64_t condition_value_size,
+    tiledb_query_condition_op_t op,
+    tiledb_query_condition_t** cond);
+
+/**
+ * Frees a TileDB query condition object.
+ *
+ * **Example:**
+ *
+ * @code{.c}
+ * uint32_t value = 5;
+ * tiledb_query_condition_t* query_condition;
+ * tiledb_query_condition_alloc(
+ *   ctx, "longitude", &value, sizeof(value), TILEDB_LT, &query_condition);
+ * tiledb_query_set_condition(ctx, query, query_condition);
+ * tiledb_query_submit(ctx, query);
+ * tiledb_query_condition_free(&query_condition);
+ * @endcode
+ *
+ * @param cond The query condition object to be freed.
+ */
+TILEDB_EXPORT void tiledb_query_condition_free(tiledb_query_condition_t** cond);
+
+/**
+ * Combines two query condition objects into a newly allocated
+ * condition. Does not mutate or free the input condition objects.
+ *
+ * **Example:**
+ *
+ * @code{.c}
+ * uint32_t value_1 = 5;
+ * tiledb_query_condition_t* query_condition_1;
+ * tiledb_query_condition_alloc(
+ *   ctx, "longitude", &value_1, sizeof(value_1), TILEDB_LT,
+ * &query_condition_1); uint32_t value_2 = 20; tiledb_query_condition_t*
```

Comment:
Can this example be broken into multiple lines? Its a bit hard to follow as is.

Reply:
Agreed, fixed. The formatter must have messed this up.